### ISSUE TITLE
Install operator failed due to webhook certificate not found

### ISSUE THREAD

**Issue by leoxhj** [Dynatrace/dynatrace-operator#778]
Hello team,

Today when we're installing dynatrace to our K8s cluster, we got following errors:

```bash
[root@localhost ~]# k logs -f dynatrace-webhook-6854fb6bf6-n6m84
{"level":"info","ts":"2022-05-09T03:00:46.002Z","logger":"dynatrace-operator-version","msg":"dynatrace-operator","version":"v0.6.0","gitCommit":"df84aed153f11165                       eef57ad7f2cab4d8b15cec51","buildDate":"2022-05-05T15:36:55+00:00","goVersion":"go1.17.9","platform":"linux/amd64"}
I0509 03:00:47.055535       1 request.go:665] Waited for 1.025999503s due to client-side throttling, not priority and fairness, request: GET:https://10.3.0.1:443                       /apis/image.toolkit.fluxcd.io/v1beta1?timeout=32s
{"level":"info","ts":"2022-05-09T03:00:47.308Z","logger":"main.controller-runtime.metrics","msg":"Metrics server is starting to listen","addr":":8383"}
{"level":"info","ts":"2022-05-09T03:00:47.308Z","logger":"main","msg":"SSL certificates configured","dir":"/tmp/k8s-webhook-server/serving-certs/","key":"tls.key                       ","cert":"tls.crt"}
{"level":"info","ts":"2022-05-09T03:00:47.308Z","logger":"main.controller-runtime.webhook","msg":"Registering webhook","path":"/label-ns"}
{"level":"info","ts":"2022-05-09T03:00:49.923Z","logger":"main.controller-runtime.webhook","msg":"Registering webhook","path":"/inject"}
{"level":"info","ts":"2022-05-09T03:00:49.923Z","logger":"main.controller-runtime.webhook","msg":"Registering webhook","path":"/livez"}
{"level":"info","ts":"2022-05-09T03:00:49.924Z","logger":"main.controller-runtime.builder","msg":"skip registering a mutating webhook, object does not implement                        admission.Defaulter or WithDefaulter wasn't called","GVK":"dynatrace.com/v1alpha1, Kind=DynaKube"}
{"level":"info","ts":"2022-05-09T03:00:49.924Z","logger":"main.controller-runtime.builder","msg":"skip registering a validating webhook, object does not implemen                       t admission.Validator or WithValidator wasn't called","GVK":"dynatrace.com/v1alpha1, Kind=DynaKube"}
{"level":"info","ts":"2022-05-09T03:00:49.924Z","logger":"main.controller-runtime.webhook","msg":"Registering webhook","path":"/convert"}
{"error":"open /tmp/k8s-webhook-server/serving-certs/tls.crt: no such file or directory","level":"error","logger":"main","msg":"problem running manager","stacktr                       ace":"github.com/go-logr/logr.Logger.Error
        /go/pkg/mod/github.com/go-logr/logr@v1.2.2/logr.go:270
main.exitOnError
        /app/src/cmd/operator/main.go:109
main.main
        /app/src/cmd/operator/main.go:95
runtime.main
{"level":"info","ts":"2022-05-09T03:00:49.924Z","logger":"main.controller-runtime.builder","msg":"Conversion webhook enabled","GVK":"dynatrace.com/v1alpha1, Kind                       =DynaKube"}
{"level":"info","ts":"2022-05-09T03:00:49.924Z","logger":"main.controller-runtime.builder","msg":"skip registering a mutating webhook, object does not implement                        admission.Defaulter or WithDefaulter wasn't called","GVK":"dynatrace.com/v1beta1, Kind=DynaKube"}
{"level":"info","ts":"2022-05-09T03:00:49.924Z","logger":"main.controller-runtime.builder","msg":"skip registering a validating webhook, object does not implemen                       t admission.Validator or WithValidator wasn't called","GVK":"dynatrace.com/v1beta1, Kind=DynaKube"}
{"level":"info","ts":"2022-05-09T03:00:49.924Z","logger":"main.controller-runtime.builder","msg":"Conversion webhook enabled","GVK":"dynatrace.com/v1beta1, Kind=                       DynaKube"}
{"level":"info","ts":"2022-05-09T03:00:49.924Z","logger":"main.controller-runtime.webhook","msg":"Registering webhook","path":"/validate"}
{"level":"info","ts":"2022-05-09T03:00:49.925Z","logger":"main","msg":"starting manager"}
{"level":"info","ts":"2022-05-09T03:00:49.925Z","logger":"main.controller-runtime.webhook.webhooks","msg":"Starting webhook server"}
{"level":"info","ts":"2022-05-09T03:00:49.925Z","logger":"main","msg":"Stopping and waiting for non leader election runnables"}
{"level":"info","ts":"2022-05-09T03:00:49.925Z","logger":"main","msg":"Stopping and waiting for leader election runnables"}
{"level":"info","ts":"2022-05-09T03:00:49.925Z","logger":"main","msg":"Stopping and waiting for caches"}
{"level":"info","ts":"2022-05-09T03:00:49.925Z","logger":"main","msg":"Starting server","path":"/metrics","kind":"metrics","addr":"[::]:8383"}
{"level":"info","ts":"2022-05-09T03:00:49.925Z","logger":"main","msg":"Stopping and waiting for webhooks"}
{"level":"info","ts":"2022-05-09T03:00:49.925Z","logger":"main","msg":"Wait completed, proceeding to shutdown the manager"}

```

once we changed back to v0.5.1, there's not such error. 

our k8s cluster is AKS, the version is 1.20.13

could you please look into this issue for your latest operator 0.6.0?

**Comment by meik99** [comment]
There is an environment variable mistakenly set in the manifests. 
Look for the environment variable 
```
- name: DEPLOYED_VIA_OLM
   value: "true"
```
in the manifest file, remove it and redeploy the operator.
It should appear twice, once for the operator deployment and once for the webhook deployment, both instances must be removed.

This issue is currently being fixed.

**Comment by leoxhj** [comment]
ok, thanks, we have deployed v0.5.1 instead, and probably will wait upgrade until its officially fix done in v0.6.0

**Comment by luhi-DT** [comment]
Hi, the fix has been officially released and you can reapply the appended yamls in the release, they have been updated correctly. Thanks for pointing this issue out to us.

Have a great day!
